ocs_ci/ocs/cluster.py

```diff
@@ -614,7 +614,7 @@ def get_rebalance_status(self):
 
         ceph_pod = pod.get_ceph_tools_pod()
         ceph_status = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph status")
-        ceph_health = ceph_status['health']['status']
+        ceph_health = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph health")
```

Comment:
why are we flipping back and forth here? last PR changed this, please run tier1 too

Reply:
@vasukulkarni Not sure if we are switching back and forth but the current logic giving TypeError hence the change is made.  I'm waiting for this test to pass in Jenkins, it's a tricky test, then I can run tier1 as well.

Reply:
@tiffanyn108 below code would get the ceph_health and I made this change in PR https://github.com/red-hat-storage/ocs-ci/pull/2316, which should solve your problem, please rebase the code and check this got merged recently.
```
        ceph_pod = pod.get_ceph_tools_pod()
        ceph_status = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph status")
        ceph_health = ceph_status['health']['status']
```

Reply:
I think even your change would do the same, only difference is below code will get output from command and the existing code would get the output from ceph status command output.
`        ceph_health = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph health")`

Reply:
@ramkiperiy sure, I will re-run test case with your latest merged of PR #2316 .

Reply:
this line should be reverted to original ?

Reply:
@vasukulkarni This is needed for query ceph_status to return a string format.  I will keep this change and tier1 tests are passing.

Reply:
@tiffanyn108 I think you missed the rebased, we are running one extra command which is fine but not required since ceph status had it.